### ISSUE TITLE
Hidden page shows up in google search

### ISSUE THREAD

**Issue by cmq-cmq** [GM-Alex/user-access-manager#82]
I am quite happy with the plugin and until now thought it does it job well. However, coincidentally, I just discovered one of my protected pages in a google search result and wonder how this could happen? Shouldn't the search-bots be prevented to access a restricted page, too?

**Comment by masimies** [comment]
Hi, lookup #78 and #57

**Comment by GM-Alex** [comment]
Are the pages completely hidden?

**Comment by cmq-cmq** [comment]
Don't understand your question, Alex. The pages are set to be accessible to certain users only.
How could I exclude these pages to be indexed (and copied with all the contents!!!) by the google-bots? And presumably other bots too (didn't research)?

**Comment by GM-Alex** [comment]
I should be mre exact. Did you choose to completely hide pages at the settings or you are show the pages but restrict the content?

**Comment by cmq-cmq** [comment]
Sorry for the delay. Yes - settings are to hide pages completely.

**Comment by GM-Alex** [comment]
Then there is no possible way that google can index them expect they where public for some time.

**Comment by GM-Alex** [comment]
@cmq-cmq Could it be that these pages which are indexed went public for some time?

**Comment by cmq-cmq** [comment]
I don't think they ever did. I have created the pages, right from the beginning with the UAM-protection. Later on, I gave edit rights to other people, but to my knowledge the UAM-Status has never been changed. Also there was no reason to do it.

The page in question is an address-listing that should be kept private within a group of users. Now the addresses show in google - although one still can't open the page with the URL provided, unless logged in and authorised. 

Quite embarrassing ...
